test_reporting/kusto/setup.kql

```diff
@@ -85,3 +85,78 @@
                                                                                   {"column":"ReportId","Properties":{"path":"$.id"}},
                                                                                   {"column":"TestbedName","Properties":{"path":"$.testbed"}},
                                                                                   {"column":"UploadTimestamp","Properties":{"path":"$.upload_time"}}]'
+
```

Comment:
If we run this setup.kql file, the new tables will be created. What about the existing tables? Will they be overwritten or just skipped?

Reply:
The data in the existing tables will not be overwritten. Having said that, we presently don't "run" this setup.kql file, we just execute these queries manually to set the tables and mappings.